db/schema.rb

```diff
@@ -12,6 +12,18 @@
 
 ActiveRecord::Schema.define(version: 20170530143955) do
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.string "commenter"
```

Comment:
I don't know why this was included. Must of been changed between me pulling and pushing?